engines/dfc_provider/app/services/quantitative_value_builder.rb

```diff
@@ -78,6 +77,8 @@ def self.apply(quantity, product)
         ["weight", "gram", 1]
       when quantity_unit.KILOGRAM
         ["weight", "kg", 1_000]
+      when quantity_unit.TONNE
+        ["weight", "kg", 1_000_000]
```

Comment:
I think this should be "T" (as in `WeightsAndMeasures`)

Reply:
Yes you are right.

And I'm just realising another issue. I chose the most accurate values for imperial measures, thinking that it can always be rounded later, but `WeightsAndMeasures` uses rounded values as lookup keys. Well, I guess, we can add the rounding to that class to help it recognise these measures.